etw/rust/tracelogging/Cargo.toml

```diff
@@ -24,8 +24,10 @@ readme = "README.md"
 rust-version = "1.63"
 
 [features]
-default = ["etw", "macros"]
+default = ["macros", "etw", "user_mode"]
 etw = [] # Logging is enabled if windows && etw.
+user_mode = []
```

Comment:
I'm not sure why we need both user_mode and kernel_mode. We should only need one new flag, `kernel_mode`.

Reply:
Removed user_mode